src/test/java/org/folio/rest/utils/UserSummaryUtils.java

```diff
@@ -0,0 +1,39 @@
+package org.folio.rest.utils;
+
+import java.util.stream.IntStream;
+
+import org.folio.domain.OpenLoan;
+import org.folio.domain.UserSummary;
+import org.folio.repository.UserSummaryRepository;
+import org.folio.rest.TestBase;
+
+import io.vertx.ext.unit.TestContext;
+
+public class UserSummaryUtils extends TestBase {
+
+  protected void checkUserSummary(String summaryId, UserSummary userSummaryToCompare,
+    TestContext context, UserSummaryRepository userSummaryRepository) {
```

Comment:
Can we make UserSummaryRepository a field in this class to avoid passing its instance into this method every time?